### ISSUE TITLE
Remove mirna from Gene

### ISSUE THREAD

**Issue by julie-sullivan** [opencb/cellbase#448]
Gene.mirna was used for 37, not used in 38. Do we still need these data? `mirna.txt` is not on the download list, so it's not being loaded anymore.

- [ ] remove mirna from GeneParser.java
- [ ] remove mirna from Gene.java in biodata

OR if it is still needed, the file should be added to the download.

**Comment by imedina** [comment]
Yes, miRNA is needed, old data sources have changed so we need to start this from scratch again. We need to look for a new data source, review data models and parse new data.

**Comment by julie-sullivan** [comment]
| Data files  | Data type |
| ------------- | ------------- |
| mirbase  | Regulation  |
| mirna.txt  | Gene  |

@imedina Just to clarify, this isn't in reference to the mirbase downloads which are in "regulation" (I haven't gotten there yet!) . I was just trying to get the "gene" build to work, and noticed there was an entry for `mirna.txt`. This file isn't downloaded, but is parsed. The parsing fails everytime obviously but a good idea to remove `mirna.txt` to prevent confusion.
